### PR TITLE
docs(blade): add basic docs to get started

### PR DESCRIPTION
**Pull request by kamleshchandnani**

### CODE REVIEW

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+<Meta title="Guides/Installation" />
+
+# ⚙️ Installation
+<br/>
+<br/>
+
+## ✍🏻 Pre-requisite
+Before you install the package, make sure that you have performed following steps:
+Before using universe just ensure following things
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Installation.stories.mdx:11]:
```suggestion
```

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+* You must be running Node version >=14.0.0
+* You must have `yarn` installed
+* Generate a Personal Access Token on GitHub by [visiting this link](https://github.com/settings/tokens/new?scopes=repo,workflow,write:packages,read:repo_hook,write:packages)
+  - From Enable SSO, click `Authorize` button next to Razorpay logo.
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Installation.stories.mdx:16]: This is a RZP specific thing and won't show up if someone from non RZP tries this.

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* Run `code ~/.bashrc` or `code ~/.zshrc` in your editor and add this line 
+  ```
+  export GITHUB_ACCESS_TOKEN="<YOUR_TOKEN>"
+  ```
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Installation.stories.mdx:20]: Add this. People tend to get confused here 😛 
``
Note: Replace `<YOUR_TOKEN>` with your actual GitHub Personal Access Token
``

File: packages/blade/docs/guides/Installation.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+* Run `code ~/.npmrc` and append the following
+```bash
+# add following to your .npmrc
+@razorpay:registry=https://npm.pkg.github.com/
+//npm.pkg.github.com/:always-auth=true
+//npm.pkg.github.com/:_authToken=${GITHUB_ACCESS_TOKEN}
+```
+## ⬇️ Add blade to your application
+```shell
+$ yarn add @razorpay/blade
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Installation.stories.mdx:31]:
```suggestion
yarn add @razorpay/blade
```

File: packages/blade/docs/guides/Usage.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,64 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Guides/Usage" />
+
+# 👀 How to use blade?
+<br/>
+<br/>
+
+## Tokens
+To start using tokens in your application you can follow below steps:
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Usage.stories.mdx:10]:
```suggestion
To start using tokens in your application you can follow these steps:
```

File: packages/blade/docs/guides/Usage.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+1. Wrap you component tree with `BladeProvider` and pass it `paymentTheme` or `bankingTheme` tokens.
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Usage.stories.mdx:11]:
```suggestion
1. Wrap your App with `BladeProvider` and pass it `paymentTheme` or `bankingTheme` tokens.
```

File: packages/blade/docs/guides/Usage.stories.mdx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+      <BladeProvider themeTokens={paymentTheme}>
+        <Card />
+      </BladeProvider>
+    </React.Fragment>
+  );
+}
+
+export default App;
+```
+2. You can also pass in optional `colorScheme` prop to the `BladeProvider` mentioning whether you want the `light`, `dark` or `system`'s default color scheme. The default is `light`.
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Usage.stories.mdx:29]:
```suggestion
2. You can also pass an optional `colorScheme` prop to the `BladeProvider` mentioning whether you want the `light`, `dark`, or `system`'s default color scheme. The default is `light`.
```

File: packages/blade/docs/guides/Usage.stories.mdx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+```jsx
+<BladeProvider themeTokens={paymentTheme} colorScheme='dark'>
+   <Card/>
+</BladeProvider>
+```
+3. After you've wrapped your component tree under `BladeProvider`, you can use the `useTheme()` hook to get the access to the current theme context.
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Usage.stories.mdx:35]:
```suggestion
3. After you've wrapped your App with `BladeProvider`, you can use the `useTheme()` hook to get access to the current theme context.
```

File: packages/blade/docs/guides/Usage.stories.mdx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+          This amount will be deducted in 3 installments from your settlement balance between Feb
+          18-20 on a daily basis.
+        </CaptionRegular>
+      </StyledCard>
+    </React.Fragment>
+  );
+};
+
+const StyledCard = styled.div(
+  ({ theme }: { theme: Theme }) => `
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Usage.stories.mdx:55]: Missing import for `Theme`?

File: packages/blade/docs/guides/Usage.stories.mdx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+    width: 368px;
+    background-color: ${theme.colors.surface.background.level2.lowContrast};
+    border-radius: ${theme.border.radius.medium}px;
+    padding: ${theme.spacing[5]}px;
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/Usage.stories.mdx:59]: Is this the correct usage for spacing? 🤔

File: packages/blade/docs/guides/LocalDevelopment.stories.mdx [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+
+<br/>
+
+## Running Tests
+We use `react-testing-library` for writing tests. If you want to write platform specific tests then suffix the test file with `filename.native.ts` or `filename.web.ts`
+
+* To run the tests for web projects follow the steps below:
+  ```shell
+  cd packages/blade
+  yarn test:web
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/LocalDevelopment.stories.mdx:45]:
```suggestion
  yarn test:react
```

File: packages/blade/docs/guides/LocalDevelopment.stories.mdx [hunk 12]
```diff
@@ -0,0 +1,52 @@
...
+  ```
+* To run the tests for native projects follow the steps below:
+  ```shell
+  cd packages/blade
+  yarn test:native
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/LocalDevelopment.stories.mdx:50]:
```suggestion
  yarn test:react-native
```

File: packages/blade/docs/guides/LocalDevelopment.stories.mdx [hunk 13]
```diff
@@ -0,0 +1,52 @@
...
+
+## Running Tests
+We use `react-testing-library` for writing tests. If you want to write platform specific tests then suffix the test file with `filename.native.ts` or `filename.web.ts`
+
+* To run the tests for web projects follow the steps below:
+  ```shell
+  cd packages/blade
+  yarn test:web
+  ```
+* To run the tests for native projects follow the steps below:
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/LocalDevelopment.stories.mdx:47]:
```suggestion
* To run the tests for react-native projects follow the steps below:
```

File: packages/blade/docs/guides/LocalDevelopment.stories.mdx [hunk 14]
```diff
@@ -0,0 +1,52 @@
...
+    # this script can vary based on the kind of project so check the example `package.json` to find the relevant script to start the project
+    yarn start
+    ```
+
+<br/>
+
+## Running Tests
+We use `react-testing-library` for writing tests. If you want to write platform specific tests then suffix the test file with `filename.native.ts` or `filename.web.ts`
+
+* To run the tests for web projects follow the steps below:
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/guides/LocalDevelopment.stories.mdx:42]:
```suggestion
* To run the tests for react web projects follow the steps below:
```

File: packages/blade/docs/tokens/Colors.stories.mdx [hunk 15]
```diff
@@ -0,0 +1,109 @@
+import { Meta, DocsContainer, Story, ColorPalette, ColorItem } from '@storybook/addon-docs';
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/tokens/Colors.stories.mdx:1]: A lot of formatting issues on this file. Can you check locally? It should use prettier here as well

File: packages/blade/docs/tokens/Colors.stories.mdx [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+  }}
+/>
+
+
+# 🎨 Color Palette
+
+## Chromatic Colors
+
+<ColorPalette>
+{Object.entries(colors.chromatic).map(([category, swatches]) => (
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/tokens/Colors.stories.mdx:42]: Some of these colors have a very horrible horizontal scroll :/

No way to make it wrap to next line?

File: packages/blade/docs/tokens/Colors.stories.mdx [hunk 17]
```diff
@@ -0,0 +1,109 @@
...
+import flat from 'flat';
+import get from 'lodash/get';
+
+<Meta
+  title="Tokens/Colors"
+  parameters={{
+    docs: {
+      container: ({ children, context }) => {
+        const getThemeTokens = () => {
+          if (context.globals.themeTokens === 'paymentTheme') {
```
Comment (chaitanyadeorukhkar) [inline packages/blade/docs/tokens/Colors.stories.mdx:13]: It's not too obvious of a change when you change the theme or scheme. For a while I thought it's not working on my local until I scrolled down to see some of the colors changing. Can we put an indicator somewhere on the top which tells you which is the current selected theme and scheme?

File: packages/blade/docs/guides/Usage.stories.mdx (hunk 10 above)
Comment (kamleshchandnani) [reply packages/blade/docs/guides/Usage.stories.mdx:59]: yes, until we add the way to automatically attach units

File: packages/blade/docs/tokens/Colors.stories.mdx (hunk 16 above)
Comment (kamleshchandnani) [reply packages/blade/docs/tokens/Colors.stories.mdx:42]: no 😢

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Pull request merged**